<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>129</red>
        <green>61</green>
        <blue>156</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>129</red>
        <green>61</green>
        <blue>156</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-image: url(:/bg/resourses/images/settings-bg.jpg);
}

QPushButton {
    font-family: 'Impact', fantasy;
    font: 15px;
    padding: 2px 2px;
    position: relative;
    border: 2px solid white;
    border-radius: 15px;
    background: rgb(187, 187, 187);
}

QPushButton:hover {
    background: rgb(226,226,226);
}


QPushButton:pressed {
    background: rgb(151,151,151);
}

QLabel {
    font-family: system-ui;
    font: 20px;
    border: 2px solid black;
    color: white;
}

QComboBox {
    font-family: system-ui;
    border: 1px solid white;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: rgb(189,189,189);
    min-width: 6em;
}

QComboBox:!editable, QComboBox::drop-down:editable {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 6px;
}

QComboBox::drop-down {
    width: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; 
}

QCheckBox {
    margin: 5px 5px 5px 0px;
 }

 QCheckBox::indicator {
     width: 13px;
     height: 13px;
 }
</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>541</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>60</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <item row="2" column="1">
     <widget class="QComboBox" name="LanguageModeButton">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Русский</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Английский</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="screenModeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Язык</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Звуки</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Полноэкранный режим</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Музыка</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="musicModeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="audio-volume-high"/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="soundModeButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="audio-volume-high"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>480</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
